Dockerfile

```diff
@@ -47,6 +47,7 @@ RUN if [ -n "${APT_PACKAGES}" ]; then apt-get update && apt-get install --no-ins
     pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu116 && \
     pip install PyYAML numpy tqdm pytorch_lightning einops numpy omegaconf && \
     pip install https://github.com/crowsonkb/k-diffusion/archive/master.zip && \
+    pip install https://github.com/AmericanPresidentJimmyCarter/xformers-builds/raw/master/cu116/xformers-0.0.14.dev0-cp310-cp310-linux_x86_64.whl && \
```

Comment:
why building custom xformers wheel and not relying on the official one ?

Reply:
The official wheel doesn't include a binaries for flash attention. I can build it inside the image but it takes about 10-20 minutes.

Reply:
Okay lets be sure we do move to the official wheel once it is out